### ISSUE TITLE
is it possible to add the icon on the right of the default value?

### ISSUE THREAD

**Issue by Vasault** [sohobloo/react-native-modal-dropdown#248]
what i mean is, i red somewhere around here that you can wrap the component and inside put a button to mimic the dropdown icon, but the thing is, this is replacing the onPress behaviour, instead of just being a single icon to differentiate between my textinputs and my dropdowns

**Comment by donni106** [comment]
Hi @Vasault, can you please create this issue in the up-to-date fork of this package here: https://github.com/siemiatj/react-native-modal-dropdown/issues?

**Comment by Vasault** [comment]
> Hi @Vasault, can you please create this issue in the up-to-date fork of this package here: https://github.com/siemiatj/react-native-modal-dropdown/issues?

will that be the main fork?

**Comment by donni106** [comment]
Yes, we try that, as discussed here: https://github.com/sohobloo/react-native-modal-dropdown/issues/251
Currently I have problems contacting @siemiatj, but there are updates in the pipeline.
Are you interested in contributing to this project @Vasault?

**Comment by Kishanjvaghela** [comment]
Try this https://github.com/Kishanjvaghela/react-native-modal-dropdown

Added support for `renderRightComponent`.

Now you can render anything in right side including icon as well.

```
<ModalDropdown
            isFullWidth
            textStyle={{ flex: 1 }}
            renderRightComponent={() => (
              <Image />
            )}
            {...rest} />
```
